WORKSPACE

```diff
@@ -1,3 +1,5 @@
+workspace(name = "org_tensorflow_serving")
```

Comment:
Talked with @damienmg and we decided to encourage but not enforce this in Bazel projects.  Changed.